qemu/tests/seabios_bin.py

```diff
@@ -30,9 +30,9 @@ def run(test, params, env):
 
     error_context.context("Get available bin files", logging.info)
     output = process.system_output('ls /usr/share/seabios', shell=True).decode()
-    bin_file_skip = params.get("bin_file_skip", "")
+    existing_bin_file = params.get("existing_bin_file", "bios.bin")
     for value in bin_dict.values():
-        if value not in output and value != bin_file_skip:
+        if value not in output and value != existing_bin_file:
```

Comment:
on rhel6, if value=bios.bin and bios.bin is not in output, this test should fail, no need to check value != existing_bin_file, when we check it the test will pass unexpected.